packages/vscode-extension/src/webview/components/AppRootSelect.tsx

```diff
@@ -16,6 +16,7 @@ function renderAppRoots(
 
   return (
     <Select.Group>
+      <Select.Label className="approot-select-label">Select project</Select.Label>
```

Comment:
I think I'd prefer
```suggestion
      <Select.Label className="approot-select-label">Select application</Select.Label>
```
But :shrug:

I think both XCode and Visual Studio (the fat one) use the terminology of "Project" being a part of "Workspace", so your version makes sense in that context.
I don't have a strong opinion on that.